nexus/resource_repository_apthosted_test.go

```diff
@@ -0,0 +1,65 @@
+package nexus
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccRepositoryAptHosted(t *testing.T) {
+	t.Parallel()
+
+	repoName := fmt.Sprintf("test-repo-%s", acctest.RandString(10))
+	repoAptDistribution := "bionic"
+	repoAptSigningKeypair := acctest.RandString(10)
+	repoAptSigningPassphrase := acctest.RandString(10)
+	repoCleanupPolicyNames := []string{"weekly-cleanup"}
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccRepositoryAptHosted(repoName, repoAptDistribution, repoAptSigningKeypair, repoAptSigningPassphrase, repoCleanupPolicyNames),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("nexus_repository.apt_hosted", "name", repoName),
+					resource.TestCheckResourceAttr("nexus_repository.apt_hosted", "format", "apt"),
+					resource.TestCheckResourceAttr("nexus_repository.apt_hosted", "type", "hosted"),
+				),
+			},
+			{
+				ResourceName:      "nexus_repository.apt_hosted",
+				ImportStateId:     repoName,
+				ImportState:       true,
+				ImportStateVerify: true,
+				// apt_signing not returned by API
```

Comment:
While you are moving things around and you are already have full overview of tests, could you please also add some notes about the current state of tests and things which are missing here:
- which fields are not being checked?
- which configurations are not being tested?

ex:
```go
// TODO: verify apt_signing configuration, apt_signing not returned by API currently
ImportStateVerifyIgnore: []string{"apt_signing"},
// TODO: add check for storage
// TODO: add check for apt.distribution
// TODO: add tests for readonly repository
```
Then later we will not forget about things to be done.